project_one.html

```diff
@@ -15,10 +15,84 @@
   </head>
   <body>
     <section class="container">
+      <div class="pageone">
+      <header class="header">
+        <nav class="logo">
+          <a href="#" class="logo1"> My Logo </a>
+          <ul>
+            <li> <a href="#">Portfolio</a> </li>
+            <li> <a href="#">About</a> </li>
+            <li> <a href="#">Contact</a> </li>
+          </ul>
+          
+          <div class="menu"><img src="./icons/Menu.png" alt="" /></div>
+        </nav>
+      </header>
+
+      <section class="content">
+        <div class="content_set" >
+        <div>
+          <h1 class="heading">I'm Saif <br> Glad to see you!</h1>
+        </div>
+        <div>
+ 
+            <p class="para_text">
+              I'm a software developer! I can help you build a product , feature
+              or website Look through some of my work and experience! If you
+              like what you see and have a project you need coded, don't
+              hestiate to contact me.
+            </p>
+       
+
+          <span class="extra">
+            <a href="#" class="connect"> LET'S CONNECT </a></span
+          >
+          <ul >
+            <li class="icons">
+              <a href="#" class="social">
+                <img
+                  src="icons/twitter.png"
+                  alt="Twitter"
+                  class="twitter"
+                />
+              </a>
+            </li>
+            <li class="icons">
+              <a href="#" class="social">
+                <img
+                  src="icons/Linkedin.png"
+                  alt="Linkedin"
+                  class="linkdein"
+                />
+              </a>
+            </li>
+            <li class="icons">
+              <a href="#" class="social">
+                <img src="icons/m.png" alt="M" class="m" />
+              </a>
+            </li>
+            <li class="icons">
+              <a href="#" class="social">
+                <img src="icons/github.png" alt="Github" class="git" />
+              </a>
+            </li>
+            <li class="icons">
+              <a href="#" class="social">
+                <img src="icons/peace.png" alt="peace" class="peace" />
```

Comment:
- [ ] Kindly fix the spacing between the social links. At the moment, there is too much space between yours when compared with the original design

### Your Design
![image](https://user-images.githubusercontent.com/43995830/160106334-1c463f63-9cdf-4fc7-8c64-7d37323823fe.png)

### Original design
![image](https://user-images.githubusercontent.com/43995830/160106440-60babe25-720b-4306-8e67-c1a86f10bb87.png)